packages/pipeline-editor/src/index.ts

```diff
@@ -154,21 +154,21 @@ const extension: JupyterFrontEndPlugin<void> = {
       label: (args: any) => {
         return args['isPalette']
           ? 'New Pipeline Editor'
-          : args.runtime?.display_name
+          : args.runtime?.title
           ? args.isMenu
-            ? `${args.runtime?.display_name} Pipeline Editor`
+            ? `${args.runtime?.title} Pipeline Editor`
```

Comment:
Just curious why "Pipeline Editor" isn't the value of `schemaType` here?  Is this not a "widget"?

Reply:
Not a big deal.  Thanks for the explanation.